### DISCUSSION TITLE
encryption status of deleted files?

### DISCUSSION THREAD

**Discussion by yipzip** [mpobaschnig/vaults#55] [Q&A] [answered]
Hello, I was looking for a solution to encrypt individual folders in Ubuntu and found Vaults. Seems to work nicely so far, thank you.

One thing that is unclear to me is whether files inside an encrypted folder remain encrypted when they are deleted (or moved out of the folder to the trash, then the trash is emptied). I need to make sure the file is in its encrypted state when deleted.

I suppose this depends in part on whether the files are decrypted to disc or to memory when mounted.

Thanks for your help!

**Comment by mpobaschnig** [comment, marked as answer]
Hey! As far as I know, FUSE-based filesystems transparently work with the files. If you delete the file in the mounted folder, the encrypted file gets deleted, just be sure that it's not moved to the trash while doing so. You might also ask the developer(s) of the filesystems for more information.

**Comment by yipzip** [comment]
Hi thank you, i should have thought about that route directly deleting.
